### DISCUSSION TITLE
Openai vision

### DISCUSSION THREAD

**Discussion by Perlsowisdom** [logancyang/obsidian-copilot#183] [Ideas]
Are there any plans to integrate Dalle3 or Openai's images API functionality. I could see this being really useful viewing a canvas and generating PowerPoint slides or visuals to highlight certain aspects of one's notes. I'm not sure how difficult that would be to add to the current plugin.

**Comment by logancyang** [comment]
That is a great direction to go. I'm thinking about it but might need more hands to carry this out.
